packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -24,7 +22,7 @@ class MockClipboard {
       case 'Clipboard.getData':
         return _clipboardData;
       case 'Clipboard.setData':
-        _clipboardData = methodCall.arguments;
+        _clipboardData = methodCall.arguments as dynamic;
```

Comment:
Do you need this cast now that the `_clipboardData` is dynamic?

Reply:
Oops, forgot to remove this when I decided to change it to dynamic